docs/source/fx.rst

```diff
@@ -298,6 +290,245 @@ Miscellanea
    -  Annotations on local names within a function are not currently
       supported.
 
+Writing Transformations
+-----------------------
+
+TODO
+
+
+Debugging
+-----------
+
+Introduction
+^^^^^^^^^^^^^^^^
+
+Once we've generated Python code from our symbolically-traced Module,
+we'll want to verify that the proper semantics were preserved. If they
+weren't, we may need to do some debugging. The key is to work
+backwards: first, check the results of the generated module, then debug
+the generated code, then debug the process of transformations that lead
+to the generated code.
+
+If you’re not familiar with debuggers, please see the auxiliary section
+:ref:`Available Debuggers`.
+
+Debugging the Generated Code
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
The lead-in mentions working backwards, but the examples in this section show directly debugging symbolic tracing. Maybe introduce `my_pass(in : nn.Module) -> nn.Module` and show that getting called, just to keep in theme with working backwards